pkg/client/errors.go

```diff
@@ -37,4 +37,5 @@ var (
 	ErrSrvIllegalArguments   = status.Error(codes.InvalidArgument, "illegal arguments")
 	ErrSrvIllegalState       = status.Error(codes.InvalidArgument, "illegal state")
 	ErrSrvEmptyAdminPassword = status.Error(codes.InvalidArgument, "Admin password cannot be empty")
```

Comment:
admin

Reply:
I didnt modified that. We will fix in a new PR